library/core/src/lib.rs

```diff
@@ -173,7 +173,7 @@
 #![feature(const_refs_to_cell)]
 #![feature(decl_macro)]
 #![feature(deprecated_suggestion)]
-#![feature(doc_cfg)]
+#![cfg_attr(bootstrap, doc_cfg)]
```

Comment:
This is what is creating the big mess. I'm finding out by what it was replaced (because we can't remove it altogether otherwise the first compilation stage will fail).